infra/openapi_specs/clu.json

```diff
@@ -25,11 +25,12 @@
           {
             "name": "api-version",
             "in": "query",
-            "description": "The API version to use for this operation. The value must be '2023-04-01'",
+            "description": "The version must be '2023-04-01'",

```

Comment:
hmm is there a way to get this to be a variable that the user injects? We may deprecate 2023-04-01 at some point

e.g. we support a bunch of different versions - https://learn.microsoft.com/en-us/rest/api/language/